packages/block-library/src/image/image-size.js

```diff
@@ -37,18 +37,17 @@ function ImageSize( { src, dirtynessTrigger, children } ) {
 				imageWidthWithinContainer: width,
 				imageHeightWithinContainer: height,
 			} );
-		};
+		}
 
+		defaultView.addEventListener( 'resize', calculateSize );
+		image.addEventListener( 'load', calculateSize );
```

Comment:
I guess there's no meaningful difference between `addEventListener( 'load', fn )` and `onload = fn` ? (Aside from the ability to have multiple event handlers in the former, which doesn't apply here anyways)

Reply:
Exactly. I like the consistency and it's clearer how the listener is removed.